src/key-derivation/pbkdf2-hmac.ts

```diff
@@ -1,3 +1,5 @@
+import { EncryptionKey } from '../encryption-key';
+import { binaryStringToBytes } from '../util';
```

Comment:
Unused import

Reply:
Linter/Prettier should complain about this anyway (if not it's a problem)

Reply:
yeah maybe it's just warning now, I noticed a bunch of unused imports when I was trying to trace usage of the new functions